packages/web-app/src/views/questionnaires-list.njk

```diff
@@ -0,0 +1,37 @@
+{% extends "layouts/main.njk" %}
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% set questionnairesListDisplayData = [] %}
+
+{% for questionnaire in sortedQuestionnairesData %}
+  {% set status %}
+      <strong class="govuk-tag govuk-tag--{{ questionnaire.status.colour }}">
+        {{ questionnaire.status.text }}
+      </strong>
+  {% endset %}
+  
+  {% set questionnairesListDisplayData = (questionnairesListDisplayData.push([
+    { html: "<a href=\"review-questionnaire-submission/" + questionnaire.id + "\">" + questionnaire.reference + "</a>" },
```

Comment:
Could we please add the `govuk-link` class to the link?

We could also consider adding the `govuk-link--no-visited-state` class as it's a caseworking system the visited state may provide no benefit to the user and potentially adds distracting colour? Perhaps a question for the design team.

Reply:
Done.

Reply:
The `govuk-link` class should be on the `a` element rather than on the parent `td` element.

Reply:
The class still appears to be on the `td` element. The class needs to be added to the link on line 9, something like 

```
{% set reference = "<a href=\"review-questionnaire-submission/" + questionnaire.id + "\" class=\"govuk-link\">" + questionnaire.reference + "</a>" %}
```

Reply:
Done.